data-and-functions-1/src/functions/getProductById.js

```diff
@@ -0,0 +1,13 @@
+import DATA from '../DATA'
+
+const getProductById = (id) => {
```

Comment:
You also need to test to make sure `data`, `data.products`, and `productId` are not null.